Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -4447,6 +4447,12 @@ enum class WebCore::MediaPlayerPitchCorrectionAlgorithm : uint8_t {
     BestForSpeech,
 };
 
+[OptionSet] enum class WebCore::MediaPlayerVideoPlaybackConfigurationOption : uint8_t {
+    Stereo,
```

Comment:
Should we have a `Mono` here? And have the concept of an empty set mean "I don't know yet"?